backend/sites/src/migrations/1727371415670-master-script.ts

```diff
@@ -5,7 +5,7 @@ export class MasterScript1727371415670 implements MigrationInterface {
 
   public async up(queryRunner: QueryRunner): Promise<void> {
     await queryRunner.query(
-      `ALTER TABLE "sites"."snapshots" DROP COLUMN "whenCreated"`,
+      `ALTER TABLE "sites"."snapshots" DROP COLUMN "when_created"`,
```

Comment:
i dont think we need to check in this change as it is required for your local machine

Reply:
this won't create issue for those who is running in docker. if somebody want to run in local machine they have to delete the migration data from table because  when_created need to be similar everywhere.

Reply:
Looking at the other tables in the database it looks like `when_created` is the existing convention so I would prefer to keep it consistent across all of the tables.

Reply:
@bolny when_created is part of BaseAuditEntity so all the entities need to extend BaseAuditEntity to keep it same.